app/components/NavigationTab/NavigationLink.js

```diff
@@ -12,9 +12,16 @@ type Props = {
 };
 
 const NavigationLink = (props: Props) => {
+  // Custom isActive function as NavLink does not compare query params by default
+  const isActive = (match, location) => {
```

Comment:
Move this function out of the component definition.

Reply:
That does not work as it requires the to prop in order to make the comparison. I could get most of the function body of the component by making it a curry function and only instantiating it inside the component, but imo that would be overkill for a function used one place.

Reply:
Ah, I didn't see the usage of `props.to`, sorry! Then this is fine